qiskit_ibm_runtime/api/rest/base.py

```diff
@@ -23,6 +23,10 @@ class RestAdapterBase:
 
     _HEADER_JSON_CONTENT = {"Content-Type": "application/json"}
 
+    _HEADER_JSON_ACCEPT = {"Accept": "application/json"}
+
+    _HEADER_API_VERSION = {"IBM-API-Version": "2024-01-01"}
```

Comment:
@jelcaf just confirming if the api version here is correct?

Reply:
Yes!